.github/workflows/tests.yml

```diff
@@ -13,14 +13,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
-        django-version: ["2.2", "3.2", "4.0"]
```

Comment:
Django versions 3.2 and 4.1 are not yet EOL, please check here: https://www.djangoproject.com/download/#supported-versions

This could break the build for many users.

Reply:
Moreover, if you change this settings, you should also change `classifiers` in `setup.cfg`.

Reply:
I removed Django 2.2 and added 4.1 and 4.2 to this workflow. I also changed setup.cfg to update the allowed Django versions.